core/controllers/dashboard.py

```diff
@@ -166,6 +167,8 @@ def _get_intro_card_color(category):
 
             feedback_thread_analytics = feedback_services.get_thread_analytics(
                 exp_summary.id)
+            exp_stats = stats_services.get_exploration_stats(
+                exp_summary.id, 'all')
             # TODO(sll): Reuse _get_displayable_exp_summary_dicts() in
```

Comment:
@526avijitgupta, can you address this at some point? It was actually intended for you to handle as part of the upcoming dashboard work.